sky/server/server.py

```diff
@@ -1023,6 +1023,7 @@ async def unzip_file(zip_file_path: pathlib.Path,
 
     def _do_unzip() -> None:
         try:
+            shutil.rmtree(client_file_mounts_dir, ignore_errors=True)
```

Comment:
There will be a race condition, if another launch is uploading the files from the same folder to the remote cluster